### PR TITLE
diffusion: fix wan2.2 ti2v num_frames adjust logic

### PR DESCRIPTION
**Pull request by mickqian**
## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/multimodal_gen/configs/pipelines/qwen_image.py [hunk 1]
```diff
@@ -4,13 +4,13 @@
 from typing import Callable
 
 import torch
+from diffusers.pipelines.qwenimage.pipeline_qwenimage_edit import calculate_dimensions
```
Comment (adarshxs) [inline python/sglang/multimodal_gen/configs/pipelines/qwen_image.py]: can you import this function from utils:
`sglang.multimodal_gen.utils import calculate_dimensions`

ref: https://github.com/sgl-project/sglang/pull/13294
